### DISCUSSION TITLE
Block or Filter a user

### DISCUSSION THREAD

**Discussion by ashfaux** [civitai/civitai#129] [Ideas]
Id like a way to block certain users so I dont see their posts or any of their uploads

**Comment by JustMaier** [comment]
Already exists :)
![image](https://user-images.githubusercontent.com/607609/211135516-51aa9f3c-3493-4239-bbef-6bf03c5ddfb8.png)

Just click on the top left of one of their model cards and click the "Hide content from this user" button. It'll make everything from them grayed out and then when you refresh they'll be gone completely.
